es7/2016-09/sept-28.md

```diff
@@ -753,7 +753,7 @@ https://github.com/domenic/proposal-import-function#an-actual-function
 
 https://github.com/domenic/proposal-import-function#a-new-binding-form
 
-DD: This is better than inserting <script type=module> tags dynamically because it's based directly on module specifiers, easier to use, doesn't pollute DOM, etc. Not introducing a new binding form.
+DD: This is better than inserting `<script type=module>` tags dynamically because it's based directly on module specifiers, easier to use, doesn't pollute DOM, etc. Not introducing a new binding form.
```

Comment:
@rwaldron  This change is important, because ...

![broken rendering](https://cloud.githubusercontent.com/assets/19714/19349981/e2b1d478-9190-11e6-9f20-a3f494a6bf5e.png)